### DISCUSSION TITLE
Opinions for Desktop Website

### DISCUSSION THREAD

**Discussion by riteshjena** [necolas/react-native-web#2537] [Q&A]
I have an app built on Native that serves my PWA in a webview and my website and PWA are built on React JS as of now.

I have been considering moving to React Native for the app. Is React native web a viable option to build desktop websites at the moment. Has anyone come across any shortcomings?

Any information about your experiences with the same would be really helpful

**Comment by necolas** [comment]
FWIW, twitter.com is a responsive web app built with React Native for Web. But if you already have a working solution using web views, it might not be worth rewriting everything.

There's also an effort underway to get React DOM code running on native, but it's not ready yet https://github.com/react-native-community/discussions-and-proposals/pull/496
